plugins/filter/ocp_compatibility.py

```diff
@@ -4,61 +4,65 @@ def filters(self):
             'ocp_compatibility': self.ocp_compatibility,
         }
 
-    def ocp_compatibility(self, after_api, curr_version, junit_ocp_file):
+    def ocp_compatibility(self, removed_in_release_api, curr_version, junit_ocp_file):
         '''
         Parse the deprecated and to-be-deprecated API after the workload installation.
         '''
         from junit_xml import TestCase, TestSuite
-        from semver import VersionInfo
 
-        def k8s2ocp(x):
+        def k8s2ocp(k8s_version):
             '''
             Convert k8s version to ocp version
             '''
-            return f"{int(x.split('.')[0]) + 3}.{int(x.split('.')[1]) - 13}"
-        # parse the API after the workload installation and write down incompatible OCP versions
-        failed_versions = {k8s2ocp(api['removedInRelease']) for api in after_api}
-        version_to_removed_apis = {
-            version: {api['name'] for api in after_api if k8s2ocp(api['removedInRelease']) == version}
-            for version in failed_versions
-        }
+            k8s_major, k8s_minor = map(int, k8s_version.split('.'))
+            ocp_major, ocp_minor = k8s_major + 3, k8s_minor - 13
```

Comment:
question: where do these magic numbers come from? just curious

Reply:
How would you convert k8s version 1.25 into OCP 4.12? I know two ways:

- major + 3, minor - 13
- float(k8s_version) + 2.87, like `round(float('1.25') + 2.87, 2)` = 4.12

We should create an official Python package :-)